### PR TITLE
Gallery - Fixing titles of some plots

### PR DESCRIPTION
**Pull request by dpshelio**
Some plots didn't have title, either because they were never set or because they margins were too big.

### CODE REVIEW

File: doc/examples/plot_adapt_rgb.py [hunk 1]
```diff
@@ -48,7 +48,9 @@ def sobel_hsv(image):
 
 fig, (ax_each, ax_hsv) = plt.subplots(ncols=2)
 ax_each.imshow(sobel_each(image))
+ax_each.set_title('Each channel separately')
 ax_hsv.imshow(sobel_hsv(image))
+ax_hsv.set_title('Value filtered channel')
```
Comment (emmanuelle) [inline doc/examples/plot_adapt_rgb.py]: How about "Filtered on HSV value channel"? I think "value" is clearer if you mention HSV as well.

**Comment by emmanuelle** [comment]
Thank you very much, titles will indeed improve the example. I just suggested a slight modification of one title for the sake of clarity.

**Comment by emmanuelle** [comment]
ping @rougier since you're working on the same example in #1659

**Comment by rougier** [comment]
Ah. What should we do ? First I commit and then we rebase this one to change the title ?

**Comment by dpshelio** [comment]
@rougier since your title (and rest of the changes for this example) are better I can just remove the changes of that title.

**Comment by emmanuelle** [comment]
OK, so first I'll merge this one to have space for title, then @rougier you can rebase and we'll merge your pull request.

**Comment by dpshelio** [comment]
@emmanuelle, @rougier does not need to rebase now because the same file has not been changed :)

**Comment by rougier** [comment]
ah too late, how do I cancel a rebase (locally and remotely) ?

**Comment by dpshelio** [comment]
@rougier  there's not problems for rebasing, you are just updating your history to the latest one.. so I don't think you need to worry.

**Pull request merged**